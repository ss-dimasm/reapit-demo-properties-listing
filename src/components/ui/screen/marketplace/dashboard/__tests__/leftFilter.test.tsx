import React from 'react'
// eslint-disable-next-line
import { shallow } from 'enzyme'
// eslint-disable-next-line
import LeftDashboardMarketPlace from '../leftDashboard'

jest.mock('../leftDashboard')

describe('Left bar', () => {
  it('should render <LeftDashboard/>', () => {
    const wrapper = shallow(
      <LeftDashboardMarketPlace
        selectedPropertyType={'Apartment'}
        marketingMode={'letting'}
        priceType={'letting'}
        changePropertyType={jest.fn()}
        changeLocalityType={jest.fn()}
        changePriceType={jest.fn()}
        changePriceRange={jest.fn()}
        changeTotalBedRoom={jest.fn()}
        clickedSearchButton={jest.fn()}
      />
    )
    expect(wrapper).toMatchSnapshot()
  })
})
