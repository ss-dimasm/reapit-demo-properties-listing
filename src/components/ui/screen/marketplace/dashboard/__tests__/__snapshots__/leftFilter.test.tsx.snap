// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Left bar should render <LeftDashboard/> 1`] = `""`;
